platforms/jvm/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junitplatform/JUnitPlatformTestFramework.java

```diff
@@ -95,9 +93,6 @@ public TestFramework copyWithFilters(TestFilter newTestFilters) {
 
     @Override
     public WorkerTestClassProcessorFactory getProcessorFactory() {
-        if (!JavaVersion.current().isJava8Compatible()) {
-            throw new UnsupportedJavaRuntimeException("Running JUnit Platform requires Java 8+, please configure your test java executable with Java 8 or higher.");
-        }
```

Comment:
Isn't this still valid or do we check somewhere else already?

Reply:
https://github.com/gradle/gradle/blob/238259ccb1a07be97d67f01b0f1fc9c77f635b95/platforms/jvm/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java#L701-L703